examples/bread/luci.snapshot.json

```diff
@@ -1,77 +1,59 @@
 {
-  "name": "bakery",
+  "name": "integration_test",
   "targets": [
     {
-      "path": "//farm:water",
+      "path": "//regular_integration_tests:text_editing",
       "buildTarget": {
-        "name": "water",
+        "name": "text_editing",
         "agentProfiles": [
-          "linux"
+          "linux", 
+          "macos", 
+          "ios", 
+          "android"
         ],
-        "runner": "Farmer"
-      }
-    },
-    {
-      "path": "//farm:seeds",
-      "buildTarget": {
-        "name": "seeds",
-        "agentProfiles": [
-          "linux"
+        dependencies: [
+          "//:flutter", //clone the correct version of flutter, enable web.
+          "//:similator_installers", // will install/validate emulators.
+          "//:web_installers", // will validate drivers and browsers. will start the drivers.
```

Comment:
This is not the right place to start the driver because the dependency may run on a different machine. Installing/validating driver installations, however, is a fair game as a dependency.